### ISSUE TITLE
Default panel LnF should follow default LnF and colourScheme global preferences

### ISSUE THREAD

**Issue by damiensellier** [damiensellier/CtrlrX#51]
Default panel LnF should follow default LnF and colourScheme global preferences : 

```
ctrlrLookAndFeel
ctrlrColourScheme

```

CtrlrPanelEditor.cpp

```
setProperty(Ids::uiPanelLookAndFeel, "V4 Light");
LookAndFeel_V4::setColourScheme(getLightColourScheme());
```

With condition defined a bit like this : 

`setProperty (Ids::ctrlrUseEditorWrapper, (bool)ctrlrPlayerInstanceTree.hasProperty(Ids::ctrlrUseEditorWrapper) ? (bool)ctrlrPlayerInstanceTree.getProperty(Ids::ctrlrUseEditorWrapper) : true);`

OR 
```

if (getInstanceMode() == InstanceSingle || getInstanceMode() == InstanceSingleRestriced) // Added v5.6.31. Force LnF V3 for Main & Child Windows if panel is Legacy
    {
        ValueTree ed = getInstanceTree().getChildWithName(Ids::uiPanelEditor);
        
        if (ed.getProperty(Ids::uiPanelLegacyMode) == "1"
            || ed.getProperty(Ids::uiPanelLookAndFeel) == "V3"
            || ed.getProperty(Ids::uiPanelLookAndFeel) == "V2"
            || ed.getProperty(Ids::uiPanelLookAndFeel) == "V1")
        {
            setProperty(Ids::uiPanelLegacyMode, "1");
            setProperty(Ids::ctrlrLookAndFeel, "V3");
        }
        else
        {
            setProperty(Ids::uiPanelLegacyMode, "0");
            setProperty(Ids::ctrlrLookAndFeel, "V4");
            
            // Requires passing the colourScheme to the property ctrlrColourScheme from uiPanelLookAndFeel
            if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 Light")
            {
                setProperty(Ids::ctrlrColourScheme, "Light");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 Grey")
            {
                setProperty(Ids::ctrlrColourScheme, "Grey");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 Dark")
            {
                setProperty(Ids::ctrlrColourScheme, "Dark");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 Midnight")
            {
                setProperty(Ids::ctrlrColourScheme, "Midnight");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 JetBlack")
            {
                setProperty(Ids::ctrlrColourScheme, "JetBlack");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 YamDX")
            {
                setProperty(Ids::ctrlrColourScheme, "YamDX");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 AkAPC")
            {
                setProperty(Ids::ctrlrColourScheme, "AkAPC");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 AkMPC")
            {
                setProperty(Ids::ctrlrColourScheme, "AkMPC");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 LexiBlue")
            {
                setProperty(Ids::ctrlrColourScheme, "LexiBlue");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 KurzGreen")
            {
                setProperty(Ids::ctrlrColourScheme, "KurzGreen");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 KorGrey")
            {
                setProperty(Ids::ctrlrColourScheme, "KorGrey");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 KorGold")
            {
                setProperty(Ids::ctrlrColourScheme, "KorGold");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 ArturOrange")
            {
                setProperty(Ids::ctrlrColourScheme, "ArturOrange");
            }
            else if (ed.getProperty(Ids::uiPanelLookAndFeel) == "V4 AiraGreen")
            {
                setProperty(Ids::ctrlrColourScheme, "AiraGreen");
            }
        }
    }
```

**Comment by damiensellier** [comment]
FIXED on 28.12.2024 for v5.6.31

![Capture d’écran, le 2024-12-28 à 19 56 39](https://github.com/user-attachments/assets/66fb4ad9-c3f0-44b6-bdc2-30377e2c6df9)
![Capture d’écran, le 2024-12-28 à 19 57 44](https://github.com/user-attachments/assets/a00b2da8-b305-4315-ad23-0f6c3b00254f)

**Comment by damiensellier** [comment]
Destructor should reset LnF to the one set in the global preferences : 

```
CtrlrPanelEditor::~CtrlrPanelEditor()
{
    //componentAnimator.removeChangeListener (this); // NOT WORKING Added back v5.6.31 for file management bottom notification bar
    getPanelEditorTree().removeListener(this);
    owner.getPanelTree().removeListener(this);
    owner.getPanelTree().removeChild(getPanelEditorTree(), 0);
    ctrlrComponentSelection->removeChangeListener(ctrlrPanelProperties);
    masterReference.clear();
    deleteAndZero(ctrlrPanelProperties);
    deleteAndZero(spacerComponent);
    deleteAndZero(ctrlrPanelViewport);
    setLookAndFeel(nullptr); // Updated v5.6.30. reset LnF to global LnF from Preferences when closing panel
}

Probably requires a list of lookandfeel schemes to apply depending on ids::ctrlrLookAndFeel & Ids::ctrlrColourScheme
```
